aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md

```diff
@@ -753,44 +753,44 @@ You can now add JavaScript files to a central list that will be implicitly inclu
 
 IntelliSense also supports title case searches:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image24.png)
+![Screenshot shows that IntelliSense also supports title case searches.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image24.png)
 
 <a id="_Toc318097417"></a>
 #### Hierarchical indentation
 
 The CSS editor uses indentation to display hierarchical rules, which gives you an overview of how the cascading rules are logically organized. In the following example, the #list a selector is a cascading child of list and is therefore indented.
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image25.png)
+![Screenshot that shows an example of the #list selector as a cascading child of list and is therefore indented.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image25.png)
 
 The following example shows more complex inheritance:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image26.png)
+![Screenshot that shows an example with more complex inheritance.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image26.png)
 
 The indentation of a rule is determined by its parent rules. Hierarchical indentation is enabled by default, but you can disable it the Options dialog box (Tools, Options from the menu bar):
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image27.png)
+![Screenshot that shows the indentation of a rule, which is determined by its parent rules. Hierarchical indentation is enabled by default, but you can disable it the Options dialog box (Tools, Options from the menu bar).](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image27.png)
 
 <a id="_Toc318097418"></a>
 #### CSS hacks support
 
 Analysis of hundreds of real-world CSS files shows that CSS hacks are very common, and now Visual Studio supports the most widely used ones. This support includes IntelliSense and validation of the star (\*) and underscore (\_) property hacks:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image28.png)
+![Screenshot shows that C S S hacks, and now Visual Studio supports the most widely used ones. This support includes IntelliSense and validation of the star and underscore  property hacks.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image28.png)
 
 Typical selector hacks are also supported so that hierarchical indentation is maintained even when they are applied. A typical selector hack used to target Internet Explorer 7 is to prepend a selector with *\*:first-child + html*. Using that rule will maintain the hierarchical indentation:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image29.png)
+![Screenshot that shows a typical selector hack used to target Internet Explorer 7 to prepend a selector with slash : first-child plus h t m l. Using this rule maintains the hierarchical indentation.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image29.png)
```

Comment:
Let's make H T M L or h t m l for the sake of consistency, I believe you used the H T M L more frequently throughout the article.